panther_lights/include/panther_lights/apa102.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PANTHER_LIGHTS_APA102_HPP_
+#define PANTHER_LIGHTS_APA102_HPP_
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace panther_lights::apa102
+{
+
+class APA102
+{
+public:
+  APA102(
+    const std::string & device, const std::uint32_t speed = 800000, const bool cs_high = false);
+  ~APA102();
+
+  void SetGlobalBrightness(const std::uint8_t brightness);
+  void SetGlobalBrightness(const double brightness);
```

Comment:
```suggestion
  void SetGlobalBrightness(const float brightness);
```
As stated everywhere else, `panther_msgs/srvs/SetLEDBrightness.data` is `float`